pkg/windows/calico.go

```diff
@@ -373,6 +374,11 @@ func (c *Calico) overrideCalicoConfigByHelm(restConfig *rest.Config) error {
 }
 
 func startFelix(ctx context.Context, config *CalicoConfig) {
+	outputFile, err := os.Create(calicoLogPath + "felix.log")
+	if err != nil {
+		logrus.Errorf("error creating felix.log: %v", err)
```

Comment:
Should we also return here?

Reply:
I am undecided here, so I would really appreciate your opinion. This is what I think: If the file does not get created, then we will get no logs. However, the process could still run successfully, as logging is not an essential part of the execution, so perhaps by just warning the user that there is a problem when creating the logging file is enough. This is actually what the code is currently doing. Are there any best practices which describe what to do in this case? Perhaps we could follow those

Reply:
Is there any reason why this can't return an error, as `startCalico() ` does? This needs to either return an error which can be handled by the caller, or it needs so `Fatalf` to avoid panicing for the reason Brian mentioned.

Reply:
thanks for the feedback. I think it makes sense

Reply:
yes. if it's a goroutine, then it shouldn't return, and should `Fatalf` on an unrecoverable error.